File: scala/CFLOW006.scala
object CFLOW006 {
  def main(args: Array[String]): Unit = {
    var z = scala.io.StdIn.readInt()
    var count = 0
    while (z > 0) {
      var a = scala.io.StdIn.readInt()
      while (a > 0) {
        var b = a % 10
        count = count + b
        a = a / 10
      }
      println(count)
      count = 0
      z = z - 1;
    }
  }
}

File: python/CFLOW006.py
z = int(input())
count = 0
while (z > 0):
    a = int(input())
    while(a > 0):
        b = a % 10
        count = count + int(b)
        a = a/10
    print(int(count))
    count = 0
    z = z-1
